sdkv2/src/main/kotlin/com/nohrd/bike/sdkv2/ByteStuffer.kt

```diff
@@ -0,0 +1,63 @@
+package com.nohrd.bike.sdkv2
+
+/**
+ * This object can be used to decode/encode messages that are sent or received from the v2 Nohrd Bike through USB
+ */
+public object ByteStuffer {
```

Comment:
Does this have to be public, is this actually something the user of the library needs to invoke?

Reply:
Writing/receiving custom packets of bluetooth is something the user of the library needs to do themselves. I figured doing the same for usb to match. This class just provides a way to encode/decode the data.